containers/ecr-viewer/src/app/tests/components/Unavailable.test.tsx

```diff
@@ -48,6 +48,16 @@ describe("UnavailableInfo", () => {
         value: "",
       },
     ];
+    const hospitalEncounterUnavailableData = [
+      {
+        title: "Hospital Admission Diagnosis",
+        value: "",
```

Comment:
I know this is following a pattern, but it would probably be a better test to actually do `evaluateHospitalEncounterData([])` and get back what the tru empty case looks like instead of hard coding here (not sure if `[]` will pass as a bundle, might need to be an object? or minimally defined `Bundle` (there are some of these kicking around elsewhere in the tests))

Reply:
Ahhhh good point, let me see what it does